README.md

```diff
@@ -1,2 +1,23 @@
-# rktlet
-The rkt implemntation of a Kubernetes Container Runtime
+# rktlet - The rkt implemntation of a Kubernetes Container Runtime
+
+The rktlet repository contains design and code related to letting the Kubelet run contains with the rkt container runtime.
+
+The work in this repository is meant to eventually supplant the [rkt package](https://github.com/kubernetes/kubernetes/tree/v1.3.0/pkg/kubelet/rkt/) in the main Kubernetes repository.
+
+## Current Status
+
+Currently the project is under design and development of the next iteration of integration between rkt and the Kubelet.
+However, the current functional integration of rkt into Kubernetes lives in the above kubelet package, not here.
+
+## Community, discussion, contribution, and support
+
+Learn how to engage with the Kubernetes community on the [community page](http://kubernetes.io/community/).
+
+You can reach the maintainers of this project at:
+
+- Slack: #sig-rktnetes
+- Mailing List: https://groups.google.com/forum/#!forum/kubernetes-sig-rktnetes
+
```

Comment:
Maybe there should be also lint to rktnetes community - https://github.com/kubernetes/community/tree/master/sig-rktnetes